example/redash_one_node/main.tf

```diff
@@ -1,27 +1,67 @@
 module "redash" {
-  source = "../../"
-  # postgres
-  postgres_service_name   = module.postgres.service_name
-  postgres_container_port = module.postgres.port
-  postgres_username       = module.postgres.username
-  postgres_password       = module.postgres.password
-
-  depends_on = [
-    module.postgres
-  ]
-}
+  source = "../.."
+  # nomad
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
 
-output "redash_username"{
-  description = "Redash admin username"
-  value       = module.redash.redash_admin_username
+  # redash
+  service         = "redash"
+  host            = "127.0.0.1"
+  port            = 5000
+  container_image = "redash/redash:9.0.0-beta.b42121"
+  use_canary      = false
+  resource_proxy = {
+    cpu    = 200
+    memory = 128
+  }
 }
 
-output "redash_password" {
-  description = "Redash admin password"
-  value       = module.redash.redash_admin_password
+
+module "redis" {
+  source = "github.com/Skatteetaten/terraform-nomad-redis.git?ref=0.1.0"
+
+  # nomad
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
+
+  # redis
+  service_name    = "redis"
+  host            = "127.0.0.1"
+  port            = 6379
+  container_image = "redis:3-alpine"
+  use_canary      = false
+  resource_proxy = {
+    cpu    = 200
+    memory = 128
+  }
+
 }
 
-output "redash_email_id" {
-  description = "Redash admin email id"
-  value       = module.redash.redash_admin_email_id
+
+module "postgres" {
+  source = "github.com/Skatteetaten/terraform-nomad-postgres.git?ref=0.4.1"
+
+  # nomad
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
+  nomad_host_volume = "persistence"
+
+  # postgres
+  service_name    = "postgres"
+  container_image = "postgres:12-alpine"
+  container_port  = 5432
+  vault_secret = {
+    use_vault_provider      = false,
+    vault_kv_policy_name    = "",
+    vault_kv_path           = "",
+    vault_kv_field_username = "",
+    vault_kv_field_password = ""
+  }
+  admin_user                      = "postgres"
+  admin_password                  = "postgres"
+  database                        = "metastore"
```

Comment:
Metastore som default databasenavn blir vel ikke riktig?

Reply:
Når jeg titter i koden til [terraform-nomad-postgres](https://github.com/Skatteetaten/terraform-nomad-postgres/blob/master/variables.tf#L72), så er `database = "metastore"` default verdi for postgres-modulen. Den brukes til å sette env-variablen `POSTGRES_DB`(https://github.com/docker-library/docs/blob/master/postgres/README.md#postgres_db). Slik jeg forstår det vil det bare bety at databasen vil bli kalt `metastore` (dersom verdien ikke er satt, så vil databasen blir kalt verdien til `POSTGRES_USER`). 

Jeg tenker at vi godt bare kan la den stå?

Reply:
Neida, dette er jo på mange måter kosemetikk. Det har ikke så mye å si at databasen til redash heter metastore.